package.json

```diff
@@ -11,6 +11,7 @@
   "main": "index.ios.js",
   "scripts": {
     "bundle": "yarn generate-graphql-query-map && react-native bundle --platform=ios --dev=false --entry-file=index.ios.js --bundle-output Pod/Assets/Emission.js --sourcemap-output Pod/Assets/Emission.js.map --assets-dest Pod/Assets && rm -rf Pod/Assets/assets/node_modules",
+    "bundle:native-ci": "yarn generate-graphql-query-map && react-native bundle --platform=ios --dev=false --entry-file=index.tests.ios.js --bundle-output Pod/Assets/Emission.js --sourcemap-output Pod/Assets/Emission.js.map --assets-dest Pod/Assets && rm -rf Pod/Assets/assets/node_modules",
```

Comment:
I see what you're doing here, and prefer it, but better to be consistent with the rest of the file and use `-`s